native~/Runtime/src/Cesium3DTilesetImpl.cpp

```diff
@@ -53,6 +53,7 @@ Cesium3DTilesetImpl::Cesium3DTilesetImpl(
 #endif
       _creditSystem(nullptr),
       _destroyTilesetOnNextUpdate(false),
+      _ctu(DotNet::CesiumForUnity::CesiumTextureUtility::Instance()),
```

Comment:
Even if we do keep the singleton around, there's no reason to store it in a field on the Cesium3DTilesetImpl. Just get it when we need it by getting the `Instance` property.

Reply:
I got it. I will change it.